Packs/CyberArk/Integrations/CyberArk/CyberArk.py

```diff
@@ -0,0 +1,934 @@
+from typing import Dict, Tuple, List
+
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+import urllib3
+import traceback
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    """
+    Client to use in the CrowdStrikeFalconX integration. Uses BaseClient
+    """
+
+    def __init__(self, server_url: str, username: str, password: str, use_ssl: bool, proxy: bool,
+                 concurrent_session: bool):
```

Comment:
maybe we can add a docstring? in general, and especially for `concurrent_session`

Reply:
yes of course we will have detailed dockstrings for all params and functions